src/SFA.DAS.AdminService.Web.Tests/Controllers/Register/RegisterBase.cs

```diff
@@ -73,12 +74,22 @@ public RegisterBase()
                 new OrganisationStandardSummary { Id = 1}
             };
 
+            List<DeliveryArea> deliveryAreas = new List<DeliveryArea>
+            {
+                new DeliveryArea { Id = 1 }
+            };
+
+            var organisationStandard = new OrganisationStandard() { OrganisationId = OrganisationOneOrganisationId };
```

Comment:
Not strictly necessary but this would make more sense if the OrganisationStandard.Id was set to organisationStandardId